src/components/Template/Template.test.tsx

```diff
@@ -242,89 +63,16 @@ describe("Template components", () => {
   it("renders the UI snapshot correctly", () => {
     const templateComponents = renderer
       .create(
-        <Template>
-          <TemplateAboveHeader>{aboveHeader}</TemplateAboveHeader>
-          <TemplateHeader>
-            {header}
-            <TemplateBreakout>{breakout}</TemplateBreakout>
-          </TemplateHeader>
-          <TemplateContent sidebar={sidebar}>
-            <TemplateContentTop>{contentTop}</TemplateContentTop>
-            <TemplateContentSidebar>{contentSidebar}</TemplateContentSidebar>
-            <TemplateContentPrimary>{contentPrimary}</TemplateContentPrimary>
-          </TemplateContent>
-          <TemplateFooter>{footer}</TemplateFooter>
+        <Template sidebar={sidebar}>
```

Comment:
Similar comment here, this is a snapshot for the left sidebar but have snapshots for other variants.